roles/federation/tasks/run_keycloak_setup.yml

```diff
@@ -0,0 +1,100 @@
+- name: Download sso76-image-stream.json
```

Comment:
This installation method seems quite brittle, is there a reason we are not just installing the operator from the operator hub like other dependent operators like nmstate and metellb: https://docs.redhat.com/en/documentation/red_hat_single_sign-on/7.6/html/server_installation_and_configuration_guide/operator#install_by_command